src/processors/default/processMempool.ts

```diff
@@ -0,0 +1,64 @@
+
+
+import _ from 'lodash';
+
+import { Table } from '../../db/db';
+import { pendingMempoolMessages } from '../../triggers/messages';
+import { CrdtOpetation, CrdtState, MempoolMessage, PendingMempoolMessage } from '../../types/message';
+import { Clients, Processor } from '../../types/processor';
+
+export const processMempool: (table: Table) => Processor = 
+  (table) => ({
+
+    name: 'processMempool',
+    trigger: pendingMempoolMessages(table),
+    processorFunction: async (messages: PendingMempoolMessage[] , clients: Clients) => {
+      // ignore inserts for now
+      const staleMessages: MempoolMessage[] = messages.filter(message => message.operation === CrdtOpetation.INSERT);
+
+      // messages are ordered by timestamp.
+      //  group by table, column, entity so that we can easily categorize fresh and stale messages
+      const groupedMessagesUpdates = _.groupBy(
```

Comment:
i'm not sure _.groupBy guarantees preservation of order? so the above assumption may be broken by it

Reply:
maybe we should sort by timestamp again just in case

Reply:
actually nvm, looks like it doesn't guarantee order of groups, but it does guarantee order of items within a group, so we should be fine